src/sentry/api/endpoints/team_time_to_resolution.py

```diff
@@ -0,0 +1,57 @@
+from collections import defaultdict
+from datetime import timedelta
+
+from django.db.models import Avg, F
+from django.db.models.functions import TruncDay
+from rest_framework.response import Response
+
+from sentry.api.base import EnvironmentMixin
+from sentry.api.bases.team import TeamEndpoint
+from sentry.api.utils import get_date_range_from_params
+from sentry.models import GroupHistory, GroupHistoryStatus, Project
+
+
+class TeamTimeToResolutionEndpoint(TeamEndpoint, EnvironmentMixin):
+    def get(self, request, team):
+        """
+        Return a a time bucketed list of mean group resolution times for a given team.
+        """
+        project_list = Project.objects.get_for_team_ids(team_ids=[team.id])
+        start, end = get_date_range_from_params(request.GET)
+        history_list = (
+            GroupHistory.objects.filter(
+                status=GroupHistoryStatus.RESOLVED,
```

Comment:
We need a way to differentiate between manual/auto resolved right?

Reply:
Right now we're just ignoring auto resolved groups, but I think if we wanted to differentiate we could query on `status=GroupHistoryStatus.AUTO_RESOLVED`